<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Untitled</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">JumpSeq-project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://han16.github.io/JumpSeq-project/">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Untitled</h1>

</div>


<!-- The file analysis/chunks.R contains chunks that define default settings
shared across the workflowr files. -->
<pre class="r"><code>#knitr::read_chunk(&quot;chunks.R&quot;)
library(knitr)
library(RColorBrewer)
library(gplots)</code></pre>
<pre><code>## KernSmooth 2.23 loaded
## Copyright M. P. Wand 1997-2009</code></pre>
<pre><code>## 
## Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:stats&#39;:
## 
##     lowess</code></pre>
<pre class="r"><code>library(ggplot2)
set.seed(123)</code></pre>
<div id="proof-of-concept" class="section level2">
<h2>Proof of concept</h2>
<div id="quantification-of-5hmc-linear-correlation-of-read-counts-vs.5hmc-amount" class="section level3">
<h3>Quantification of 5hmC: linear correlation of read counts vs.Â 5hmC amount</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-2-1.png" alt=" Fig 1.2: The fitted regression line (red) on the fold change of number of reads (point) at different hmC proportions in spike in" width="672" />
<p class="caption">
Fig 1.2: The fitted regression line (red) on the fold change of number of reads (point) at different hmC proportions in spike in
</p>
</div>
</div>
<div id="spatial-distribution" class="section level3">
<h3>Spatial distribution</h3>
<p>The reference is GSM882244_FDR_0.0484. First got non-overlapping 5hmC regions and make sure each window has only one 5hmC site. For each window, count the reads depth at every site, then add up reads depth of these non-overlapping windows. Reads distribution is the accumulative reads depth at every base of all non-overlapping windows. Input Bam file is the replicate with largest number of reads.</p>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-3-1.png" width="672" /><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-3-2.png" width="672" /></p>
</div>
<div id="correlation-among-replicates" class="section level3">
<h3>Correlation among replicates</h3>
</div>
<div id="corrleation-among-replicates" class="section level3">
<h3>corrleation among replicates</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-4-1.png" alt="Reads correlations in 1kb windows between two 5hmC replicates. log2 transformation in calculation of correlations  of number of reads" width="672" />
<p class="caption">
Reads correlations in 1kb windows between two 5hmC replicates. log2 transformation in calculation of correlations of number of reads
</p>
</div>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-7-1.png" width="672" /></p>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-8-1.png" width="672" /></p>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
</div>
<div id="comparison-of-jump-vs-nano-seal-with-strong-tab-seq-as-gold-standard" class="section level3">
<h3>Comparison of Jump vs nano-seal with strong TAB-seq as gold standard</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<ul>
<li>JumpSeq: 1kb windows are called at FDR 0.05. To make a fair comparison, we trim the jump-seq peak windows to be of the same length as nano_seq peaks.</li>
</ul>
</div>
<div id="enrichments" class="section level3">
<h3>Enrichments</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-11-1.png" alt="For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb. " width="672" />
<p class="caption">
For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb.
</p>
</div>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-11-2.png" alt="For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb. " width="672" />
<p class="caption">
For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb.
</p>
</div>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-11-3.png" alt="For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb. " width="672" />
<p class="caption">
For jump-seq, we called 1kb windows at FDR0.05 as peaks and computed the proportion of peaks (in red) overlapping with Nano-seal peaks and Tab-seq peaks. Randomly pick same number of 1kb windows in the whole genome as jump-seq peaks at different DNA levels and calculate its proportion of windows (in blue) overlapping with Nano-seal peaks and Tab-seq peaks extended to 1kb.
</p>
</div>
<ul>
<li>Jump-seq; 1kb window are called at FDR 0.05</li>
</ul>
</div>
<div id="sensitvity" class="section level3">
<h3>Sensitvity</h3>
<table>
<caption>Number of Tab-seq peaks recovered by Jump_seq and nano-seal</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">0.6ng</th>
<th align="right">2.4ng</th>
<th align="right">6ng</th>
<th align="right">12ng</th>
<th align="right">24ng</th>
<th align="right">48ng</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Jump_seq</td>
<td align="right">832</td>
<td align="right">1116</td>
<td align="right">19686</td>
<td align="right">219644</td>
<td align="right">649783</td>
<td align="right">715179</td>
</tr>
<tr class="even">
<td>Nano_seal (50ng)</td>
<td align="right">465501</td>
<td align="right">465501</td>
<td align="right">465501</td>
<td align="right">465501</td>
<td align="right">465501</td>
<td align="right">465501</td>
</tr>
</tbody>
</table>
</div>
<div id="enrichment-in-genomic-features" class="section level3">
<h3>Enrichment in genomic features</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-14-1.png" alt=" Eleven 48 ng samples are combined together and 20 bp peak windows are called at FDR 0.05. Random windows are randomly selected  in the whole genome with the same amount of peak windows. The enrichments (compare Jump-Seq peaks with random peaks) in  different genomic regions are calculated" width="672" />
<p class="caption">
Eleven 48 ng samples are combined together and 20 bp peak windows are called at FDR 0.05. Random windows are randomly selected in the whole genome with the same amount of peak windows. The enrichments (compare Jump-Seq peaks with random peaks) in different genomic regions are calculated
</p>
</div>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-15-1.png" alt="For 0.6ng 5hmC sample, 20 bp windows with at least one reads are called peak windows. Random windows are chosen in the whole genome in the same amount of peak windows. The enrichments (Jump-seq peaks compared with random windows) different genomic regions (200bp bins) are computed. " width="672" />
<p class="caption">
For 0.6ng 5hmC sample, 20 bp windows with at least one reads are called peak windows. Random windows are chosen in the whole genome in the same amount of peak windows. The enrichments (Jump-seq peaks compared with random windows) different genomic regions (200bp bins) are computed.
</p>
</div>
<ul>
<li>cis-regulatory regions are extended 200bp windows.</li>
<li>jump peaks are 20 bp windows with at least one reads, enrichment is compared with random windows with the same size.</li>
</ul>
<pre class="r"><code>getwd()</code></pre>
<pre><code>## [1] &quot;/project/G/Jump-Seq_manuscript/JumpSeq-project/analysis&quot;</code></pre>
<p>To be fair, 1kb Jump-sea peaks are trimed to be of the same length as nano-seal peaks.</p>
</div>
</div>
<div id="pn-analysis" class="section level2">
<h2>PN analysis</h2>
<div id="pn-enrichment-in-different-genomic-features" class="section level3">
<h3>PN enrichment in different genomic features</h3>
<pre class="r"><code>library(tidyr)</code></pre>
</div>
<div id="hmc-maternal-rep1-fold-of-enrichment" class="section level3">
<h3>5hmc Maternal Rep1, fold of enrichment</h3>
<pre class="r"><code>ggplot(data_rep1_maternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() + theme(axis.text.x = element_text(size =8, color = &quot;black&quot;, vjust = 1), axis.text.y = element_text(size =8, color = &quot;black&quot;))</code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-20-1.png" width="672" /></p>
</div>
<div id="hmc-maternal-rep2" class="section level3">
<h3>5hmc Maternal Rep2</h3>
<pre class="r"><code>#data_rep2_maternal$reads.norm1 &lt;- data_rep2_maternal$reads/data_rep2_maternal$remove_PCR_duplicates
ggplot(data_rep2_maternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() + theme(axis.text.x = element_text(size =8, color = &quot;black&quot;, vjust = 1), axis.text.y = element_text(size =8, color = &quot;black&quot;))</code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-22-1.png" width="672" /></p>
</div>
<div id="hmc-paternal-rep1" class="section level3">
<h3>5hmc Paternal Rep1</h3>
<pre class="r"><code># data_rep1_paternal$reads.norm1 &lt;- data_rep1_paternal$reads/data_rep1_paternal$remove_PCR_duplicates
ggplot(data_rep1_paternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() + theme(axis.text.x = element_text(size =8, color = &quot;black&quot;, vjust = 1), axis.text.y = element_text(size =8, color = &quot;black&quot;))</code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-24-1.png" width="672" /></p>
</div>
<div id="hmc-paternal-rep2" class="section level3">
<h3>5hmc Paternal Rep2</h3>
</div>
<div id="mc-maternal-rep1-fold-of-enrichment" class="section level3">
<h3>5mc Maternal Rep1, fold of enrichment</h3>
<pre class="r"><code>ggplot(data_rep1_maternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() </code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-28-1.png" width="672" /></p>
</div>
<div id="mc-maternal-rep2" class="section level3">
<h3>5mc Maternal Rep2</h3>
<pre class="r"><code>#data_rep2_maternal$reads.norm1 &lt;- data_rep2_maternal$reads/data_rep2_maternal$remove_PCR_duplicates
ggplot(data_rep2_maternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() </code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-30-1.png" width="672" /></p>
</div>
<div id="mc-paternal-rep1" class="section level3">
<h3>5mc Paternal Rep1</h3>
<pre class="r"><code># data_rep1_paternal$reads.norm1 &lt;- data_rep1_paternal$reads/data_rep1_paternal$remove_PCR_duplicates
ggplot(data_rep1_paternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() + theme(axis.text.x = element_text(size =8, color = &quot;black&quot;, vjust = 1), axis.text.y = element_text(size =8, color = &quot;black&quot;))</code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-32-1.png" width="672" /></p>
</div>
<div id="mc-paternal-rep2" class="section level3">
<h3>5mc Paternal Rep2</h3>
<pre class="r"><code># data_rep2_paternal$reads.norm1 &lt;- data_rep2_paternal$reads/data_rep2_paternal$remove_PCR_duplicates
ggplot(data_rep2_paternal, aes(x = Sample_Note, y = Enrichment, group = annotation, col = annotation)) + geom_line() + theme(axis.text.x = element_text(size =8, color = &quot;black&quot;, vjust = 1), axis.text.y = element_text(size =8, color = &quot;black&quot;))</code></pre>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-34-1.png" width="672" /></p>
</div>
</div>
<div id="enrichment-in-different-annotations-of-up-regulateddown-regulated-genes-comparing-2-cell-stage-and-mo." class="section level2">
<h2>Enrichment in different annotations of up-regulated/down-regulated genes comparing 2-cell stage and MO.</h2>
</div>
<div id="combine-a-few-conditions-across-replicatesion." class="section level2">
<h2>combine a few conditions across replicatesion.</h2>
<pre><code>## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated</code></pre>
<div id="enrichment-in-genes-up-regulated-in-2-cell-stage-compared-to-mo" class="section level3">
<h3>Enrichment in genes up-regulated in 2-cell stage compared to MO</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-37-1.png" alt="Enrichment in genes up-regulated in 2-cell stage compared to MO" width="672" />
<p class="caption">
Enrichment in genes up-regulated in 2-cell stage compared to MO
</p>
</div>
</div>
<div id="enrichment-in-genes-down-regulated-in-2-cell-stage-compared-to-mo" class="section level3">
<h3>Enrichment in genes down-regulated in 2-cell stage compared to MO</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-38-1.png" alt="Enrichment in genes down-regulated in 2-cell stage compared to MO" width="672" />
<p class="caption">
Enrichment in genes down-regulated in 2-cell stage compared to MO
</p>
</div>
</div>
</div>
<div id="enrichment-in-different-annotations-of-top-5000-expressedbottom-5000-expressed-genes-at-the-2-cell-stage" class="section level2">
<h2>Enrichment in different annotations of top 5000 expressed/bottom 5000 expressed genes at the 2-cell stage</h2>
</div>
<div id="combine-a-few-conditions-across-replicatesion.-1" class="section level2">
<h2>combine a few conditions across replicatesion.</h2>
<pre><code>## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated

## Warning in `[&lt;-.factor`(`*tmp*`, iseq, value = &quot;combine&quot;): invalid factor
## level, NA generated</code></pre>
<div id="enrichment-in-top-5000-expressed-genes-at-the-2-cell-stage" class="section level3">
<h3>Enrichment in top 5000 expressed genes at the 2-cell stage</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-41-1.png" alt="Enrichment in top 5000 expressed genes at the 2-cell stage" width="672" />
<p class="caption">
Enrichment in top 5000 expressed genes at the 2-cell stage
</p>
</div>
</div>
<div id="enrichment-in-bottom-5000-expressed-genes-at-the-2-cell-stage" class="section level3">
<h3>Enrichment in bottom 5000 expressed genes at the 2-cell stage</h3>
<div class="figure">
<img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-42-1.png" alt="Enrichment in bottom 5000 expressed genes at the 2-cell stage" width="672" />
<p class="caption">
Enrichment in bottom 5000 expressed genes at the 2-cell stage
</p>
</div>
</div>
</div>
<div id="compare-jump-seq-with-nano-hmc-seal" class="section level2">
<h2>Compare Jump-Seq with Nano-hmc-Seal</h2>
<div id="rep-6ng-vs-1000-cell-nano-sealdedup-merged-from-two-biological-replicates-10kb-window" class="section level3">
<h3>160402 rep, 6ng vs 1000-cell nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-43-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.8685951</code></pre>
</div>
<div id="rep-6ng-vs-1000-cell-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-1" class="section level3">
<h3>160511 rep, 6ng vs 1000-cell nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-44-1.png" width="672" /></p>
<pre><code>##          [,1]
## [1,] 0.862631</code></pre>
</div>
<div id="rep-6ng-vs-1000-cell-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-2" class="section level3">
<h3>160521 rep, 6ng vs 1000-cell nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-45-1.png" width="672" /></p>
<pre><code>##          [,1]
## [1,] 0.867717</code></pre>
</div>
<div id="rep-6ng-vs-1000-cell-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-3" class="section level3">
<h3>160604 rep, 6ng vs 1000-cell nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-46-1.png" width="672" /></p>
<pre><code>##          [,1]
## [1,] 0.862498</code></pre>
</div>
<div id="rep-6ng-vs-1000-cell-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-4" class="section level3">
<h3>160620 rep, 6ng vs 1000-cell nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-47-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.8707161</code></pre>
</div>
<div id="rep-48ng-vs-50ng-nano-sealdedup-merged-from-two-biological-replicates-10kb-window" class="section level3">
<h3>160402 rep, 48ng vs 50ng nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-48-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.9160376</code></pre>
</div>
<div id="rep-48ng-vs-50ng-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-1" class="section level3">
<h3>160511 rep, 48ng vs 50ng nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-49-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.9160376</code></pre>
</div>
<div id="rep-48ng-vs-50ng-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-2" class="section level3">
<h3>160521 rep, 48ng vs 50ng nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-50-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.9234346</code></pre>
</div>
<div id="rep-48ng-vs-50ng-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-3" class="section level3">
<h3>160604 rep, 48ng vs 50ng nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-51-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.9111268</code></pre>
</div>
<div id="rep-48ng-vs-50ng-nano-sealdedup-merged-from-two-biological-replicates-10kb-window-4" class="section level3">
<h3>160620 rep, 48ng vs 50ng nano-seal(dedup, merged from two biological replicates), 10kb window</h3>
<p><img src="181223_FinalAnalysis_updated_files/figure-html/unnamed-chunk-52-1.png" width="672" /></p>
<pre><code>##           [,1]
## [1,] 0.9190462</code></pre>
</div>
</div>

<hr>
<p>
    This <a href="http://rmarkdown.rstudio.com">R Markdown</a> site was created with <a href="https://github.com/jdblischak/workflowr">workflowr</a>
</p>
<hr>

<!-- To enable disqus, uncomment the section below and provide your disqus_shortname -->

<!-- disqus
  <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'rmarkdown'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
    <a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
-->


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
